modules/registry-accessing-metrics.adoc

```diff
@@ -18,12 +18,47 @@ endpoint.
 
 .Procedure
 
-There are two ways in which you can access the metrics, running a metrics query
-or using the cluster role.
+You can access the metrics by running a metrics query using a cluster role.
+
+*Cluster role*
+
+. Create a cluster role if you do not already have one to access the metrics:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
```

Comment:
I think the render would look better if the sample output yaml block was separate from the command block. Similar to this: 

```
[source,terminal]
----
<command> 
----

.Example output
[source,yaml] 
----
<yaml>
----
```